docs/src/views/FocusTrapZone.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react'
+import { Header } from '@stardust-ui/react'
+import DocPage from '../components/DocPage'
+import { link, code } from '../utils/helpers'
+
+import CodeSnippet from '../components/CodeSnippet'
+
+export default () => (
+  <DocPage title="Focus Trap Zone">
+    <Header as="h2">Content</Header>
+    <ul>
+      <li>{link('Overview', '/focus-trap-zone#overview')}</li>
+      <li>{link('Usage', '/focus-trap-zone#usage')}</li>
+      <li>
+        {link(
+          'Override FocusTrapZone settings',
+          '/focus-trap-zone#override-focustrapzone-settings',
+        )}
+      </li>
+    </ul>
+    <Header as="h2">Overview</Header>
+    <p>
+      FocusTrapZone grabs the focus and traps it within an HTML element, usually a dialog or popup.
+      Pressing TAB key will circle focus within the inner focusable elements of the FocusTrapZone.
+      The main purpose is to block user interaction outside FocusTrapZone in any way. Therefore,
+      keyboard events are not propagated outside FocusTrapZone.
+    </p>
+    <p>
+      Stardust leverages Focus Trap Zone component which is based on the{' '}
+      {link(
+        'Focus Trap Zone from Office UI Fabric.',
+        'https://developer.microsoft.com/en-us/fabric#/components/focustrapzone',
+      )}
+    </p>
+    <Header as="h2">Usage</Header>
+    <p>
+      In Stardust focus trap is applied through accessibility behavior, as for focus zone. To enable
+      focus trap for component, it is needed, in behavior, to set prop {code('trapFocus')} to{' '}
+      {code('true')} with default settings or set an object with desired values for focus trap zone
+      props. {link('Read more about Accessibility Behaviors.', '/accessibility-behaviors')}
+      Currently, it is used for Popup via {code('popupFocusTrapBehavior')} and Dialog via{' '}
+      {code('dialogBehavior')}.
+    </p>
+    <p>FocusTrapZone's props which can be applied in accessibility behavior:</p>
```

Comment:
actually, this might bring an maintainability problems to us, as here, essentially, we are copying the API and making no guarantees that it would stay up to date with real code, as we have for all our components' APIs. Is there any better way to handle that - maybe introduce link here to the API that is guaranteed to reflect latest changes in code?

Reply:
Only if to introduce link directly to GitHub's source code with `FocusZone`'s types https://github.com/stardust-ui/react/blob/master/packages/react/src/lib/accessibility/FocusZone/FocusZone.types.ts